pytests/test_network_create_args.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from podman_compose import get_network_create_args
+
+class TestGetNetworkCreateArgs(unittest.TestCase):
+    def test_minimal(self):
+        net_desc = {
+            "labels": [],
+            "internal": False,
+            "driver": None,
+            "driver_opts": {},
+            "ipam": {"config": []},
+            "enable_ipv6": False
+        }
+        proj_name = "test_project"
+        net_name = "test_network"
+        expected_args = [
+            "create",
+            "--label", f"io.podman.compose.project={proj_name}",
+            "--label", f"com.docker.compose.project={proj_name}",
+            net_name
+        ]
+        args = get_network_create_args(net_desc, proj_name, net_name)
+        self.assertEqual(args, expected_args)
+
+    def test_with_options(self):
+        net_desc = {
+            "labels": ["label1", "label2"],
+            "internal": True,
+            "driver": "bridge",
+            "driver_opts": {"opt1": "value1", "opt2": "value2"},
+            "ipam": {
+                "driver": "default",
+                "config": [{"subnet": "192.168.0.0/24", "ip_range": "192.168.0.2/24", "gateway": "192.168.0.1"}]
+            },
+            "enable_ipv6": True
+        }
+        proj_name = "test_project"
+        net_name = "test_network"
+        expected_args = [
+            "create",
+            "--label", f"io.podman.compose.project={proj_name}",
+            "--label", f"com.docker.compose.project={proj_name}",
+            "--label", "label1",
+            "--label", "label2",
+            "--internal",
+            "--driver", "bridge",
+            "--opt", "opt1=value1",
+            "--opt", "opt2=value2",
+            "--ipam-driver", "default",
+            "--ipv6",
```

Comment:
Would be great to have a separate test just for ipv6. It's hard to understand unit test when it's large.

In case you have time it makes sense to split other options into separate tests too, but I'm happy that you added any kind of tests for untested code in the first place :) For the PR only test for `--ipv6` is necessary in a strict sense.

Reply:
Thanks.